testsuite/tests/ui/conftest.py

```diff
@@ -188,7 +211,7 @@ def _custom_ui_tenant(username: str = "",
 
         account = resilient.resource_read_by_name(master_threescale.accounts, organisation)
         tenant = master_threescale.tenants.read(account.entity_id)
-        tenant.wait_tenant_ready()
+        _ui_wait_tenant_ready(tenant)
```

Comment:
What happens when `_ui_wait_tenant_ready(tenant)` returns False?

Reply:
Will continue anyways. 
I put assert before it, so it wont happen.

Reply:
Changed logic of the function. Now raises `AssertationError` if the backoff fails on last try.

Reply:
Reverted the True/False logic, now it uses logging tool to log reasons for backoff. Tell me if its good idea.